### ISSUE TITLE
--resume

### ISSUE THREAD

**Issue by yinyuhangi** [ultralytics/yolov5#1436]
## ❔Question
If I train 100 Epochs the first time.Then, looking at the results, I found no convergence. How to train 200 epochs on the basis of 100 Epochs? Will the loss curve and mAP in the resulting diagram follow the original 100 Epochs?

**Comment by glenn-jocher** [comment]
@yinyuhangi you train normally starting from your pretrained weights:

python train.py --weights pretrianed/weights.pt
